test/junit/scala/collection/immutable/LazyListTest.scala

```diff
@@ -101,12 +101,12 @@ class LazyListTest {
 
   @Test // scala/bug#9134
   def filter_map_properly_lazy_in_tail: Unit = {
-    assertLazyListOpLazyInTail(_.filter(_ % 2 == 0).map(identity), List(1, 2))
+    assertLazyListOpLazyInTail(_.filter(_ % 2 == 0).map(identity), List(1))
```

Comment:
IIUC this means that when we call `filter` and `map` we evaluate no elements of the lazy list, whereas we previously evaluated the first element?

Reply:
correct